helper/schema/schema_test.go

```diff
@@ -3947,6 +3947,33 @@ func TestSchemaMap_Validate(t *testing.T) {
 			Err: false,
 		},
 
+		"Good sub-resource, interpolated value": {
```

Comment:
Since we're already in here, I think a test with a computed value (that fails before the IsComputed check) would be useful too.